pipeline/talend_test/ansible/schema_restore.yaml

```diff
@@ -0,0 +1,35 @@
+- name: create schema download directory
+  delegate_to: localhost
+  file:
+    path: /tmp/talend_test_schemas
+    state: directory
+  when: schema_restore is defined
```

Comment:
Not required?  This set of actions isn't called unless its defined  (more below)

Reply:
thanks, forgot that I had refactored these tasks into a new playbook. Just force-pushed an update